cmd/commands/subvolume.go

```diff
@@ -0,0 +1,41 @@
+package command
+
+import (
+	"github.com/rook/kubectl-rook-ceph/pkg/subvolume"
+	"github.com/spf13/cobra"
+)
+
+// SubvolumeCmd represents the rook command to manage subvolumes
```

Comment:
```suggestion
```

I think we can remove this comment as from the code and `Short` section it is pretty clear.